lib/src/widgets/choice_button.dart

```diff
@@ -1,41 +1,73 @@
+import 'dart:ui';
+
 import 'package:cap_countdown/src/exam/optional_question.dart';
 import 'package:cap_countdown/src/exam/question_choice.dart';
+import 'package:cap_countdown/src/util/events_enum.dart';
 import 'package:flutter/material.dart';
 import 'package:latext/latext.dart';
 
 class ChoiceButton extends StatelessWidget {
   final QuestionChoice choice;
   final OptionalQuestion question;
   final bool submitted;
+  final bool isCrossOut;
   final ValueChanged<QuestionChoice?>? onChanged;
+  final ValueChanged<EventsEnum?>? onEvent;
 
   const ChoiceButton(
       {super.key,
       required this.choice,
       required this.question,
       required this.submitted,
-      this.onChanged});
+      required this.isCrossOut,
+      this.onChanged,
+      this.onEvent});
 
   @override
   Widget build(BuildContext context) {
-    return RadioListTile<QuestionChoice>(
-      // Use LaTexT to render LaTeX (math formula) in text.
-      title: LaTexT(
-          laTeXCode:
-              Text('(${choice.answer.name}) ${choice.description ?? ''}')),
-      value: choice,
-      groupValue: question.selectedChoice,
-      fillColor: _getFillColor(),
-      onChanged: (value) {
-        // If the question has been submitted, can't change the answer.
-        if (submitted) return;
-
-        if (value == question.selectedChoice) {
-          onChanged?.call(null);
-        } else {
-          onChanged?.call(value);
+    final text = '(${choice.answer.name}) ${choice.description ?? ''}';
+
+    return GestureDetector(
+      onHorizontalDragEnd: (DragEndDetails dragEndDetails) {
+        if (dragEndDetails.velocity.pixelsPerSecond.dx < 0) {
+          onEvent?.call(EventsEnum.crossOutChoice);
         }
```

Comment:
```suggestion
        final offset = dragEndDetails.velocity.pixelsPerSecond.dx;

        if (offset != 0) {
          onEvent?.call(EventsEnum.crossOutChoice);
        }
```

有沒有考慮改成這樣？ 左右滑動都有效果，我覺得這樣子比較好用。

Reply:
感覺可以